app/src/organisms/ChooseRobotSlideout/AvailableRobotOption.tsx

```diff
@@ -7,57 +7,93 @@ import {
   TYPOGRAPHY,
   SIZE_1,
 } from '@opentrons/components'
+import { Trans, useTranslation } from 'react-i18next'
 import { StyledText } from '../../atoms/text'
 import { MiniCard } from '../../molecules/MiniCard'
 import OT2_PNG from '../../assets/images/OT2-R_HERO.png'
+import { NavLink } from 'react-router-dom'
 
 interface AvailableRobotOptionProps {
   robotName: string
   robotModel: string
   local: boolean | null
   onClick: () => void
   isSelected: boolean
+  isOnDifferentSoftwareVersion: boolean
 }
 
 export function AvailableRobotOption(
   props: AvailableRobotOptionProps
 ): JSX.Element {
-  const { robotName, robotModel, local, onClick, isSelected } = props
+  const {
+    robotName,
+    robotModel,
+    local,
+    onClick,
+    isSelected,
+    isOnDifferentSoftwareVersion,
+  } = props
+  const { t } = useTranslation('protocol_list')
   return (
-    <MiniCard onClick={onClick} isSelected={isSelected}>
-      <img
-        src={OT2_PNG}
-        css={css`
-          width: 6rem;
-        `}
-      />
-      <Flex
-        flexDirection={DIRECTION_COLUMN}
-        marginLeft={SPACING.spacing4}
-        marginTop={SPACING.spacing3}
+    <>
+      <MiniCard
+        onClick={onClick}
+        isSelected={isSelected}
+        isNonviable={isOnDifferentSoftwareVersion}
       >
-        <StyledText as="h6">{robotModel}</StyledText>
-        <Box maxWidth="9.5rem">
-          <StyledText
-            as="p"
-            css={{ 'overflow-wrap': 'break-word' }}
-            fontWeight={TYPOGRAPHY.fontWeightSemiBold}
-          >
-            {robotName}
-            <Icon
-              // local boolean corresponds to a wired usb connection
-              aria-label={local ? 'usb' : 'wifi'}
-              marginBottom={`-${SPACING.spacing2}`}
-              marginLeft={SPACING.spacing3}
-              name={local ? 'usb' : 'wifi'}
-              size={SIZE_1}
-            />
-          </StyledText>
-        </Box>
-      </Flex>
-    </MiniCard>
+        <img
+          src={OT2_PNG}
+          css={css`
+            width: 6rem;
+          `}
+        />
+        <Flex
+          flexDirection={DIRECTION_COLUMN}
+          marginLeft={SPACING.spacing4}
+          marginTop={SPACING.spacing3}
+        >
+          <StyledText as="h6">{robotModel}</StyledText>
+          <Box maxWidth="9.5rem">
+            <StyledText
+              as="p"
+              css={{ 'overflow-wrap': 'break-word' }}
+              fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+            >
+              {robotName}
+              <Icon
+                // local boolean corresponds to a wired usb connection
+                aria-label={local ? 'usb' : 'wifi'}
+                marginBottom={`-${SPACING.spacing2}`}
+                marginLeft={SPACING.spacing3}
+                name={local ? 'usb' : 'wifi'}
+                size={SIZE_1}
+              />
+            </StyledText>
+          </Box>
+        </Flex>
+        {isOnDifferentSoftwareVersion ? (
```

Comment:
```suggestion
       {isOnDifferentSoftwareVersion && isSelected ? (
```

without isSelected
![Screen Shot 2022-06-24 at 2 52 35 PM](https://user-images.githubusercontent.com/474225/175648814-a8560502-7c69-4b4d-a7fc-d64a2ad826cd.png)

Reply:
Actually, the alert icon is rendering but there isn't enough space.

![Screen Shot 2022-06-24 at 3 05 01 PM](https://user-images.githubusercontent.com/474225/175649195-37067fed-cb77-4000-85c1-5ec8d43f233e.png)